.circleci/config.yml

```diff
@@ -44,6 +36,7 @@ commands:
             echo 'export LIBRA_DUMP_LOGS=1' >> $BASH_ENV
             echo 'export CARGO_INCREMENTAL=0' >> $BASH_ENV
             echo 'export CI_TIMEOUT="timeout 40m"' >> $BASH_ENV
+            echo 'export RUST_NIGHTLY=nightly-2020-03-18' >> $BASH_ENV
```

Comment:
Would it be easier to set `nightly-2020-03-18` as CircleCI env var?  For reference, https://circleci.com/gh/libra/libra/edit#env-vars

Reply:
Could be, but I think it's better to keep it in config.yml in case new changes require new nightly version. More people have access to config.yml than to circleci settings.